ui/appui-layout-react/src/appui-layout-react/widget/ContentRenderer.tsx

```diff
@@ -77,13 +78,17 @@ export function WidgetContentRenderer(props: WidgetContentRendererProps) {
       }
     };
   }, [renderTo, widgetContentManager, props.tabId]);
+  // TODO: If problems are found when using theme options such as high contrast
+  // in a separate window. That will need to be added here.
   return ReactDOM.createPortal(
     <div
       style={{ height: "100%", width: "100%" }}
       id={`content-container:${props.tabId}`}
     >
       <TabIdContext.Provider value={props.tabId}>
-        {props.children}
+        <ThemeProvider style={{ height: "100%" }}>
```

Comment:
(For history: We removed the ThemeProvider from the automatic ContentRenderer because it causes popouts to be contained within floating widgets, which is a change of behavior in floating widget that we dont want to force at this point) Filed #612 to address this.